### ISSUE TITLE
Logs Popover Menu automatic-popup UX is painful

### ISSUE THREAD

**Issue by pnepywoda** [grafana/grafana#86081]
Feature was added in https://github.com/grafana/grafana/pull/77616.

Automatically popping up a dialogue when selecting any text in logs is normally NOT what I intend to do - usually I'm copying text to put elsewhere, in JIRA for investigation, or in another loki dashboard that takes arguments. If I miss-select, the dialogue gets in the way of expanding my selection. If I immediately click away from the dialogue, it pops back up where my cursor is.

Could this become a right-click action instead?

**Comment by zizzpudding** [comment]
@matyax  I'm not sure we override the browser's right click anywhere else in Grafana. I wonder if we need to consider another way of solving this?

**Comment by gtk-grafana** [comment]
As a note, this can be disabled with the `logRowsPopoverMenu` feature flag.

**Comment by tylercrumpton** [comment]
I do like the ability to add selected text as filters quickly, but also agree that the automatic behavior can often get in the way of just selecting normally for copying. One thing that helps make this a little easier to use is that hitting ESC closes the popover without deselecting your text.

**Comment by matyax** [comment]
> One thing that helps make this a little easier to use is that hitting ESC closes the popover without deselecting your text.

Good catch! We specifically added it to quickly get it out of your way when you don't need it.

**Comment by matyax** [comment]
We're going to add a menu option to disable this feature.
